Makefile

```diff
@@ -220,3 +220,54 @@ PHONY: .shell
 		-w /work \
 		--user $$(id -u):$$(id -g) \
 		$(BUILDIMAGE)
+
+BATS_IMAGE = batstests:$(GIT_COMMIT_SHORT)
+
+.PHONY: bats-image
+bats-image:
+	docker buildx build . -t $(BATS_IMAGE) -f ci-tests.Dockerfile
+
+TEST_CHART_REPO ?= "oci://ghcr.io/nvidia/k8s-dra-driver-gpu"
+TEST_CHART_VERSION ?= $(VERSION_GHCR_CHART)
+TEST_CHART_LASTSTABLE_REPO ?= "oci://ghcr.io/nvidia/k8s-dra-driver-gpu"
+TEST_CHART_LASTSTABLE_VERSION ?= "25.3.2-7020737a-chart"
+
+# Currently consumed in upgrade test via
+# kubectl apply -f <URL> (can be a branch, tag, or commit)
+TEST_CRD_UPGRADE_TARGET_GIT_REF ?= $(GIT_COMMIT_SHORT)
+
+# Warning: destructive against currently configured k8s cluster.
+#
+# Explicit invocation of 'cleanup-from-previous-run.sh' (could also be done as
+# suite/file 'setup' in bats, but we'd lose output on success). During dev, you
+# may want to add --show-output-of-passing-tests (and read bats docs for other
+# cmdline args).
+.PHONY: bats-tests
+bats-tests: bats-image
```

Comment:
`make bats-tests`? `make bats`? Something completely different? I happily take naming suggestions. For now, I'd like to not use `make test`, but something slightly more specific. I can foresee that we will always have multiple test suites, and this here is a specific one.

Reply:
See my suggestion above

Reply:
moved to tests/bats, implemented top-level `make bats`